packages/phalanx/src/Phalanx_MDField.hpp

```diff
@@ -665,4 +665,15 @@ namespace PHX {
 
 }
 
+namespace Kokkos{
+  template <typename ...Args>
+  const auto create_mirror_view(const PHX::MDField<Args...> &a) { return Kokkos::create_mirror_view(a.get_static_view()); }
+
+  template <typename KokkosType, typename ...Args>
+  void deep_copy(const KokkosType& k, const PHX::MDField<Args...> &a) { Kokkos::deep_copy(k, a.get_static_view()); }
+
+  template <typename KokkosType, typename ...Args>
+  void deep_copy(const PHX::MDField<Args...> &a, const KokkosType& k) { Kokkos::deep_copy(a.get_static_view(), k); }
+}
```

Comment:
@rppawlo thoughts on augmenting the kokkos namespace and not just using as_view() at the point of usage?

Reply:
Ok, I reverted the changes. Let's be consistent with using `as_view`